tmt/cli.py

```diff
@@ -614,6 +614,11 @@ def tests_import(
     '--ignore-git-validation', is_flag=True,
     help="Ignore unpublished git changes and export to Nitrate. "
     "The case might not be able to be scheduled!")
+@click.option(
+    '--append-summary / --no-append-summary', default=False,
+    help="When set the Nitrate case summary '{repository.name} {test.name}' has "
+    "' - {test.summary}' appended to it."
+    )
```

Comment:
The help message is a bit hard to read. What about this?

    Include test summary in the Nitrate/Polarion test case summary as well.
    By default, only the repository name and test name are used.

Polarion should be mentioned too as the new option has been added there as well, right?

Reply:
yeah it probably should, even though for Polarion this is a bit redundant anyway as summary is already used as title for the case

Reply:
Thanks for the suggestion, used as is in c0f6c9c